corehq/apps/app_manager/views/view_generic.py

```diff
@@ -303,7 +306,27 @@ def _make_name(suffix):
     confirm = request.session.pop('CONFIRM', False)
     context.update({'confirm': confirm})
 
+    if show_app_version_changes_notification(request, domain):
```

Comment:
I think having this in the AppSummaryView is probably a better spot. Doesn't this show on like every app page otherwise?

Reply:
Although I 100% agree with you (I purposefully made my first commit to have this in the AppSummaryView), I made a 2nd commit where I moved it out to the [main view](https://staging.commcarehq.org/a/charlsmit-testy/apps/view/951ed40d7142426fc86137c7b367ddf8/) for the app (based on [this comment](https://dimagi-dev.atlassian.net/browse/SC-91?focusedCommentId=16071)).

If you think we can leave it as per first commit, I feel comfortable reverting and leaving it at that.

Reply:
Weird. I think we should leave it as is in the first commit.